bundles/org.openhab.binding.omnilink/README.md

```diff
@@ -0,0 +1,218 @@
+# HAI/Leviton OmniLink Binding
+
+This binding integrates the [OmniPro and Lumina](http://www.leviton.com/en/products/security-automation/automation-av-controllers/omni-security-systems) line of home automation systems. At Its core the OmniPro is a hardware board that provides security and access features. It connects to many other devices through serial ports or wired contacts and exposes them through a single TCP based API.
+
+## Supported Things
```

Comment:
Can you add a table with the Channels? You could take a look at other bindings for some examples.